services/frontend-service/src/ui/NavigationBar/NavigationBar.scss

```diff
@@ -13,6 +13,20 @@
   }
 
   .mdc-drawer__content {
-    height: 100%;
+    height: 95%;
```

Comment:
Why is this at `95%`? is it because of the top bar? or?
Can you move it into a variable and define it in https://github.com/freiheit-com/kuberpult/blob/main/services/frontend-service/src/assets/_variables.scss?

Reply:
Also Please Move the css for the indicator and for the list-item-container into a new scss file in the navigation directory: 
https://github.com/freiheit-com/kuberpult/tree/main/services/frontend-service/src/ui/components/navigation
`/navListItem.scss` for example.

Reply:
I placed that value at 95% because of the the width of the indicator. This to avoid that any overflow would occur

Reply:
I fixed it according to your suggestion!